### ISSUE TITLE
[bug]: When I try to run init on a project with Tailwind V4 alpha I get an error

### ISSUE THREAD

**Issue by fernandortec** [shadcn-ui/ui#3318]
### Describe the bug

When I have the alpha version of tailwind (v4.0.0) and try to add shadcn-ui by running init to any project I get this error.
The only solution is to add it manually. I followed [this](https://tailwindcss.com/blog/tailwindcss-v4-alpha) guide to add Tailwind V4

`error: Tailwind CSS is not installed. Visit https://tailwindcss.com/docs/installation to get started.
      at /private/tmp/bunx-501-shadcn-ui@latest/node_modules/shadcn-ui/dist/index.js:3:7112`to add

### Affected component/components

initialization of project

### How to reproduce

1. Create a React Project (Vite or NextJs it doesn't matter, they both throw the same error)
2. Add the latest alpha versions of Tailwind and PostCSS ` npm install tailwindcss@next @tailwindcss/postcss@next`
3. Run `npx shadcn-ui@latest init`

### Codesandbox/StackBlitz link

https://codesandbox.io/p/devbox/dazzling-cache-vrthst?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cluhikr9800073b6euygyy6i6%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cluhikr9800023b6e7d0mqglj%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cluhikr9800043b6eyz6p1ipd%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cluhikr9800063b6e1s4g7oxz%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cluhikr9800023b6e7d0mqglj%2522%253A%257B%2522id%2522%253A%2522cluhikr9800023b6e7d0mqglj%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522cluhikr9800063b6e1s4g7oxz%2522%253A%257B%2522id%2522%253A%2522cluhikr9800063b6e1s4g7oxz%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A3000%252C%2522id%2522%253A%2522cluhilint00dg3b6e0bvyhhel%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%2522%257D%255D%252C%2522activeTabId%2522%253A%2522cluhilint00dg3b6e0bvyhhel%2522%257D%252C%2522cluhikr9800043b6eyz6p1ipd%2522%253A%257B%2522id%2522%253A%2522cluhikr9800043b6eyz6p1ipd%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cluhil1us004x3b6euk6u3p3l%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522cluhil1yf000wd9gee96h78zl%2522%257D%255D%252C%2522activeTabId%2522%253A%2522cluhil1us004x3b6euk6u3p3l%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

### System Info

```bash
Mac Os 2021 M1
Bun 1.1 (package manager I'm using for this project)
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by TvojTatk0** [comment]
I have very same issue but using tailwind 3.4.3 in remix application. 
`pnpm dlx shadcn-ui@latest init` ends up with same error

**Comment by fernandortec** [comment]
> I have very same issue but using tailwind 3.4.3 in remix application. `pnpm dlx shadcn-ui@latest init` ends up with same error
> 
> EDIT: I had to first instal `tainwind` and create basic `tailwind.config.ts` in order to run `init` without error.

yes installing tailwind and adding the config file will most likely solve the problem for versions below 4.0.0, but the v4 alpha doesn't have any config file, it doesn't recognize tailwind.config.ts

**Comment by kerryj89** [comment]
Tailwind v4 beta dropped which no longer seems to be using `@tailwind` directive nor `tailwind.config.js` so some code will need to be updated/rewritten if the project decides to adopt Tailwind 4.

https://github.com/search?q=repo%3Ashadcn-ui%2Fui+%40tailwind&type=code
https://github.com/search?q=repo%3Ashadcn-ui%2Fui+tailwind.config.js&type=code

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
